### ISSUE TITLE
What privileges are required for the Elasticsearch user?

### ISSUE THREAD

**Issue by dmgeurts** [domainaware/parsedmarc#441]
Trying to get the direct connection going to an existing ELK stack and getting a 403 error. So I know this issue, but haven't found a list of privileges.

**Comment by AnaelMobilia** [comment]
Hello @dmgeurts ,

[Based on the source code](https://github.com/domainaware/parsedmarc/blob/95954c5d870a296fb0459abbfdb966700dcac3d9/parsedmarc/elastic.py#L6), it looks like parsedmarc need to :
- Manage indexes (create, update)
- Perform search
- Save document

If never you found the required privileges, please let the information as we can update the doc (or you can made a PR if you prefer :-)).

Regeards

**Comment by dmgeurts** [comment]
Hi @AnaelMobilia ,

Thank you, the following is what I started with. I'm still pretty new to Elasticsearch and we're running v8 here.
<img width="1479" alt="image" src="https://github.com/domainaware/parsedmarc/assets/2071992/776eb796-bc37-4491-a709-139753d8054f">

I'm not sure if the indice is even right...

I did notice that the python code for elasticsearch in the venv does cater to api_key, which would be the preferred way to authenticate rather than uid/pwd. But that's a later concern to me.
